### PR TITLE
[WIP][ADD] Backport functionality from 12.0 to web_keyboard_navigation module.

### PR DESCRIPTION
**Pull request by daramousk**

### CODE REVIEW

**Comment by sudhir-erpharbor** [comment]
This is a nice feature to have it in v11. I tried this module and test it in firefox and chrome. It works only when you pass `debug=assets`, but does not work normally.

@daramousk 
CC: @pedrobaeza

File: web_keyboard_navigation/README.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    :target: https://runbot.odoo-community.org/runbot/162
+163
+164
+186
+210
+225
+258/11.0
+
+
+Known issues / Roadmap
```
Comment (nikul-serpentcs) [inline web_keyboard_navigation/README.rst:38]: @daramousk If not need `Known issues / Roadmap` fragment then remove it

File: web_keyboard_navigation/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Copyright 2018 Therp BV <https://therp.nl>
```
Comment (nikul-serpentcs) [inline web_keyboard_navigation/__init__.py:1]: @daramousk IMO remove copyright from all `__init__` file

File: web_keyboard_navigation/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "web_keyboard_navigation",
+    "version": "11.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
```
Comment (nikul-serpentcs) [inline web_keyboard_navigation/__manifest__.py:6]: @daramousk Add `website` key as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Comment by sudhir-erpharbor** [comment]
@daramousk Any update on this?

**Comment by daramousk** [comment]
Ignore my previous edit, just a misguided debug session.

In any case please do test again. @sudhir-erpharbor

**Pull request closed without merging**